src/runtime/deps.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ActivityPorts} from '../components/activities';
+import {Config} from '../api/subscriptions';
+import {DialogManager} from '../components/dialog-manager';
+import {Doc} from '../model/doc';
+import {JsError} from './jserror';
+import {PageConfig} from '../model/page-config';
+
+export interface DepsDef {
```

Comment:
Should we rename this to Deps? I think elsewhere we've been renaming Def from the name since we no longer need to bypass the jslint for unused types.

Reply:
For sure, updated